torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -279,6 +279,8 @@ class FullyShardedDataParallel(nn.Module):
             the near future. It allows users to enable two different backward_prefetch
             algorithms to help backward communication and computation overlapping.
             Pros and cons of each algorithm is explained in the class ``BackwardPrefetch``.
+        ignored_modules (Optional[Iterable[torch.nn.Module]]): Modules whose
+            parameters are ignored by this instance. (Default: ``None``)
```

Comment:
Maybe add some details on what it means to be ignored (i.e. not sharded, not managed by FSDP).